packages/tjdashboard/administrator/sql/uninstall.mysql.utf8.sql

```diff
@@ -0,0 +1,2 @@
+DROP IF EXISTS `#__tj_dashboards`;
```

Comment:
@monikaatvowel Check this with vaishali

cc
@twsvaishali

Reply:
removed the fields now using params